assets/src/edit-story/elements/video/test/output.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import VideoOutput from '../output';
+
+describe('Video output', () => {
+  it('should produce valid AMP output', async () => {
+    const props = {
+      element: {
+        id: '123',
+        type: 'video',
+        mimeType: 'video/mp4',
+        poster: 'https://example.com/poster.png',
+        scale: 1,
+        origRatio: 9 / 16,
+        x: 50,
+        y: 100,
+        height: 1920,
+        width: 1080,
+        rotationAngle: 0,
+        resource: {
+          type: 'video',
+          mimeType: 'video/mp4',
+          videoId: 123,
+          src: 'https://example.com/image.png',
+          height: 1920,
+          width: 1080,
+        },
+      },
+      box: { width: 1080, height: 1920, x: 50, y: 100, rotationAngle: 0 },
+    };
+
+    await expect(<VideoOutput {...props} />).toBeValidAMP();
```

Comment:
Trying to figure out where in this system the `<script custom-element="amp-video">` is inserted before testing.

Reply:
The AMP Optimizer in the matcher would add any missing extensions.

Reply:
Reflect this now in the docblock for `getAMPValidationErrors`